index.html

```diff
@@ -1,138 +1,110 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="styles.css">
-
-</head>
-<body>
-    <h1>"Hello World, we are software developers!"</h1>
-    <link
-  rel="stylesheet"
-  href="https://use.fontawesome.com/releases/v5.8.1/css/all.css"
-  integrity="sha384-50oBUHEmvpQ+1lW4y57PTFmhCaXp0ML5d60M1M7uH2+nqUivzIebhndOJK28anvf"
-  crossorigin="anonymous"
-/>
-<div id="page-wrapper">
-  <header id="header">
-    <div class="logo">
-      <img
-        id="header-img"
-      />
-    </div>
-
-    <nav id="nav-bar">
-      <ul>
-        <li><a class="nav-link" href="#features">Features</a></li>
-        <li><a class="nav-link" href="#how-it-works">How It Works</a></li>
-        <li><a class="nav-link" href="#pricing">Pricing</a></li>
-      </ul>
-    </nav>
-  </header>
-
-  <div class="container"></div>
-
-  <section id="hero">
-    <h2>Handcrafted, home-made masterpieces</h2>
-    <form id="form">
-      <input
-        name="email"
-        id="email"
-        type="email"
-        placeholder="Enter your email address"
-        required
-      />
-      <input id="submit" type="submit" value="Get Started" class="btn" />
-    </form>
+<main id="main">
+  <h1 id="title">Dr. Norman Borlaug</h1>
+  <p>The man who saved a billion lives</p>
+  <figure id="img-div">
+    <img
+      id="image"
+      src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+      alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists"
+    />
+    <figcaption id="img-caption">
+      Dr. Norman Borlaug, third from the left, trains biologists in Mexico on
+      how to increase wheat yields - part of his life-long war on hunger.
+    </figcaption>
+  </figure>
+  <section id="tribute-info">

```

Comment:
Nice using Section tag, i think its more professional to put it inside a body tag